src/_posts/2024-06-14-Tales-From-TCamp.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: blog_post
+title: "Tales from Transportation Camp"
+subtitle: "Lessons learned from 2024 LA Transportation Camp."
+description: "Lessons learned from 2024 LA Transportation Camp."
+author: Vyki Englert, Scott Frazier, Nina Dinh, Marie Araneta
+excerpt: "Lessons learned from 2024 LA Transportation Camp."
+date: 2024-06-13 19:03:13 +0200
+categories: compiler
+---
+
+Recently, four members of the Compiler team had the opportunity to attend the latest edition of [UCLA’s Transportation Camp LA](https://www.its.ucla.edu/). Shout out
+to Juan Matute and the wonderful team at UCLA Institute for Transportation Studies for bringing together folks from advocacy, academia, and both the public and private sectors on a Saturday
+to explore what transit means to us and our broader community. Each of our team members got to participate in a variety of breakouts and we wanted to share our experiences.
+
+## <img src="/src/assets/team_members/nina-dinh.png" class="align-center w-25" alt="Watercolor illustration of Nina Dinh"> Nina
```

Comment:
It appears this image isn't showing, so I believe you have to remove the `/src/` as the image for Marie appears to be working

Reply:
Hm, so odd...I went back and forth between /src/ and no /src/ and it finally worked when I removed /src/ for all but Marie but it doesn't appear to be working anymore. I went ahead and removed it again but it's not showing up for me. Perhaps @angela-tran might have an idea?

Reply:
wait....now its showing up. @AnthonyRollins does it look right to you?